action.yaml

```diff
@@ -0,0 +1,28 @@
+name: Pull Request Milestoner
+description: Adds milestones to pull requests based on the base branch
+branding:
+  icon: map
+  color: blue
+inputs:
+  token:
+    description: The GITHUB_TOKEN secret
+    required: true
+
+runs:
+  using: composite
+  steps:
+    - name: Setup Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: '3.x'
+
+    - name: Get the PyGithub module
+      run: pip install PyGithub
+      shell: bash
+
+    - name: Add Milestone to the PR (if needed)
+      run: ${{ github.action_path }}/github-milestoner.py
+      shell: bash
```

Comment:
Just curious: what is this `shell` line for?  Is it needed?

Reply:
That was a holdover from the [pr-labeler](https://github.com/open-mpi/pr-labeler/blob/main/action.yaml). 

The [default shell](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepsshell) for Linux runners is `bash`, so we should be able to safely remove both this one and on the `pip install PyGithub`. Unless there's a reason we need to specify `bash`.

Reply:
Well that was a lie. So actually the `shell` option is [required](https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#runsstepsshell) if `run` is set.